packages/syntax-floating-components/src/Popover/Popover.module.css

```diff
@@ -0,0 +1,7 @@
+.popoverContent {
+  box-sizing: border-box;
+  background-color: var(--color-base-white);
+  padding: 28px;
+  border-radius: 24px;
+  width: 352px;
```

Comment:
Ideally I would like to pass in the width I want. But one thing to note this will be annoying on devices smaller than 352px. If we are forced to set a width i think it makes sense to set this to maxWidth of 352px. Just so if it needs to shrink down it can. But again ideally would like to pass in my own widths and maxWidths lol.

Reply:
maybe atleast make lg md sm size popovers or something lol.

Reply:
oh we can get rid of sizes, we discussed that with aj yesterday. ill get rid of it now

Reply:
itll take in a width that you can pass in

Reply:
maxWidth and width field please